presto/client.py

```diff
@@ -349,9 +349,10 @@ def post(self, sql, additional_http_headers=None):
                 location = http_response.headers["Location"]
                 url = self._redirect_handler.handle(location)
                 logger.info(
-                    "redirect {} from {} to {}".format(
-                        http_response.status_code, location, url
-                    )
+                    "redirect {} from {} to {}",
```

Comment:
That's definitely a good improvement!

nit: do you think it would be ok to format this in one line
```
logger.info("redirect {} from {} to {}", http_response.status_code, location, url)
```
?

(BTW i think this could go at debug level, but that's perhaps out of scope of this PR)

Reply:
Yeah of course I can format it in one line :)